docs/deployment/helm.md

```diff
@@ -213,6 +213,7 @@ API's you need to install the prometheus operator in your cluster.
 | `master.readinessProbe.periodSeconds`       | integer | 10 (by Kubernetes)               | Specifies how often (in seconds) to perform the readiness probe.                                                                                                                                      |
 | `master.readinessProbe.timeoutSeconds`      | integer | 1 (by Kubernetes)                | Specifies the number of seconds after which the probe times out.                                                                                                                                      |
 | `master.readinessProbe.successThreshold`    | integer | 1 (by Kubernetes)                | Specifies the number of consecutive successes of readiness probes before considering the pod as ready.                                                                                                |
+| `master.dnsPolicy`                          | array   | ClusterFirstWithHostNet                               | NFD master pod [dnsPolicy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy)                                                                                 |
```

Comment:
Thank you @killianmuldoon for the improvement.

One nit: could you align the columns in the tables in the docs? We've been trying to keep these in shape. Makes reviews and reading the raw table easier.

```suggestion
| `master.dnsPolicy`                          | array   | ClusterFirstWithHostNet          | NFD master pod [dnsPolicy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy)                                                                                 |
```

Reply:
Sure thing 😄